app/views/components/_quick-escape-banner.html.erb

```diff
@@ -3,6 +3,12 @@
   aria_label ||= "Hide this page. Quick exit button, it opens BBC Weather in a new tab"
   data_attributes ||= {}
   data_attributes[:id] = "app-quick-escape-banner-link"
+
+  onboarding_title ||= "What the ‘hide this page’ button does"
```

Comment:
Does this need to be user configurable? I feel like this the aria_label and the link_label all present easy opportunities for things to go wrong and that, unless we discovered evidence customisation made sense, should be kept static. For example it wouldn't make sense for users if this component was used in multiple places with different link test.

Viewing the component guide shows how easy it is to get this wrong currently:

![Screenshot 2021-04-22 at 12 09 19](https://user-images.githubusercontent.com/282717/115707299-7fbcbf80-a366-11eb-88e5-af6de5862ee6.png)